sys/include/senml.h

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ * Copyright (C) 2016 Lennart Dührsen <lennart.duehrsen@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+/**
+ * @ingroup     senml
+ * @{
+ *
+ * @file
+ * @brief       Reference implementation of the Sensor Markup Language (SenML)
+ *
+ * The Sensor Markup Language is a specification that defines media types for
+ * representing simple sensor measurements and device parameters.
+ * Representations are defined in JavaScript Object Notation (JSON), Concise
+ * Binary Object Representation (CBOR), eXtensible Markup Language (XML), and
+ * Efficient XML Interchange (EXI), which share the common SenML data model.
+ * This implementation supports only JSON and CBOR. A simple sensor, such as a
+ * temperature sensor, could use this media type in protocols such as HTTP or
+ * CoAP to transport the measurements of the sensor or to be configured.
+ *
+ * @see <a href="https://tools.ietf.org/html/draft-ietf-core-senml-04">
+ *      Media Types for Sensor Markup Language (SenML)
+ *      </a>
+ *
+ * @author      Lennart Dührsen <lennart.duehrsen@fu-berlin.de>
+ */
+
+
+#ifndef SENML_H
+#define SENML_H
+
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+
+#define SENML_SUPPORTED_VERSION (5)    //!< The highest SenML version this implementation supports
```

Comment:
Here and in other places: Please use C-style comments `/* */` (`/**< */` for inline doxygen ;-))